### ISSUE TITLE
Issues with links between pages in VoiceOver

### ISSUE THREAD

**Issue by DavidBiddle** [XGovFormBuilder/digital-form-builder#702]
**Describe the bug**
The polylines that represent links between pages in the visualisation have 2 closely related issues in VoiceOver:
- They can be selected via Voiceover's navigation, but only respond to click events (so the user can't press Enter to select them)
- They aren't currently labelled - this means all of the links just read 'Image' in a screen reader and the user doesn't know that they're interactive controls

**To Reproduce**

1. Open any form with at least one link in the designer
2. Use a screen reader to navigate through the items in the visualisation
3. Note that the links don't indicate that they're interactive controls, or give any contextual information about which pages they link

**Expected behavior**
In a screen reader the links should be labelled appropriately so that they give:
- an indication of their interactivity
- information about which pages they connect

**Screenshots**
n/a

**Repository**

- designer

**Desktop (please complete the following information):**
I believe this will happen in any screen reader, but have observed it in:

- OS: macOS
- Browser: Safari 
- Version: 15.1
- Screen reader:  VoiceOver

- **Smartphone (please complete the following information):**

- n/a

**Additional context**
I think implementing #491 would be the best fix for this, but that looks like a significantly bigger bit of work. 

I've pushed a rough [fix for this in a temporary branch in our fork](https://github.com/davidbiddle/digital-form-builder/commit/9015e536797f417541c4266d19f0fc2672d7ba1b) - happy to raise a PR once I've written a test and cleaned it up a bit. This will also fix #701

**Comment by jenbutongit** [comment]
Thank you so much @DavidBiddle!! #491 of the many things I haven't been able to get to but thought a lot about. I'll update my musings over on the issue - you might find interesting/want to implement yourself for CIFU. Anyway, I had a little scroll through, LGTM so far
